prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+const person = {
+  firstName: 'Julie',
+  lastName: 'Dumbrique',
+  hobby: 'playing cozy games on my switch'
+};
+
+console.log(person);
+
+const fullName = 'Julie ' + 'Dumbrique.';
```

Comment:
Hi Julie! To get the first and last name of the person object, you can use "dot notation".

```js
var fullName = person.firstName + ' ' + person.lastName;
```